near-sdk-sim/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "near-sdk-sim"
+version = "0.1.0"
+authors = ["Illia Polosukhin <illia.polosukhin@gmail.com>"]
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+codegen-units = 1
+# Tell `rustc` to optimize for small code size.
+opt-level = "z"
+lto = true
+debug = false
+panic = "abort"
+overflow-checks = true
+
+[dependencies]
+serde_json = "*"
+serde = { version = "*", features = ["derive"] }
+near-sdk = { path = "../near-sdk", version = "2.0.0"}
+# near-sdk = "1.0.0"
+quickcheck = "0.9"
+quickcheck_macros = "0.9"
+log = "0.4"
+env_logger = { version = "0.7.1", default-features = false }
```

Comment:
It is quite undesirable to have crates in near-sdk-rs that were not published.

Also, you might be able to compile near-crypto to wasm or near-runtime-standalone to Wasm.

In general, I am not sure how this is supposed to work and why this Cargo.toml is configured for Wasm compilation.

Reply:
Ah my bad it was a copy and paste error.